Bookish/Bookish.Web/Views/Home/Catalogue.cshtml

```diff
@@ -42,15 +42,20 @@
     </table>
 </div>
 
-<div class="padded-top btn-group" role="group">
-    <a type="button" class="btn btn-outline-dark @Model.GetDisabledClass(false)" 
-       asp-action="Catalogue" 
-       asp-route-pageNumber="@(Model.PageNumber - 1)"
-       asp-route-searchTerm="@Model.SearchTerm">Previous</a>
-    <a type="button" class="btn btn-outline-dark disabled">@Model.PageNumber</a>
-    <a type="button" class="btn btn-outline-dark @Model.GetDisabledClass(true)" 
-       asp-action="Catalogue" 
-       asp-route-pageNumber="@(Model.PageNumber + 1)"
-       asp-route-searchTerm="@Model.SearchTerm">Next</a>
-</div>
+<nav>
```

Comment:
If all the results fit on one page then I'd expect the pagination buttons to be hidden 🙂